nima/webclient/webclient/src/main/java/io/helidon/nima/webclient/http1/ClientRequestImpl.java

```diff
@@ -332,7 +333,7 @@ private ClientResponseImpl invokeServices(WebClientService.Chain callChain,
                                                                         query,
                                                                         UriFragment.empty(),
                                                                         headers,
-                                                                        Context.create(),
+                                                                        Contexts.context().orElse(Context.create()),
```

Comment:
please use `orElseGet(Context::create)` - so the context does not get created even when it exists

Reply:
Should this be configurable with a builder method ? That would be on par with 3.x.